olwebsite-app/src/components/Header.js

```diff
@@ -1,17 +1,29 @@
-import React from 'react';
-import homeLogo from '../images/homeLogo.png';
+import React from "react";
+import { Link, withRouter } from "react-router-dom";
+
+import homeLogo from "../images/homeLogo.png";
 
 function Header() {
   return (
     <header>
-      <img src={homeLogo} alt="logo" tabIndex={0} />
-      <div tabIndex={0}>Meet the team</div>
-      <div tabIndex={0}>DevDocs</div>
-      <div tabIndex={0}>Blogs</div>
-      <div tabIndex={0}>Resources</div>
-      <div tabIndex={0}>Login</div>
+      <Link to="/">
+        <img className="logo" src={homeLogo} alt="one leif logo" />
+      </Link>
+      <Link className="link" to="/Meet-the-team">
```

Comment:
maybe a more specific classname? like top-nav-link? If we get more hierarchical in the future this should probably be a standard of sorts

Reply:
General Link should be fine... CSS is cascading so we should have a link then also a nav-link for any differences...